mangaki/mangaki/utils/anidb.py

```diff
@@ -93,7 +93,7 @@ def lang_map(self) -> Dict[str, ExtLanguage]:
 
     @cached_property
     def unknown_language(self) -> ExtLanguage:
-        return ExtLanguage.objects.get(source='anidb', ext_lang='x-unk')
+        return ExtLanguage.objects.filter(source='anidb', ext_lang='x-unk').first()
```

Comment:
The issue was that if there's no result, then an error is raised : `mangaki.models.DoesNotExist: ExtLanguage matching query does not exist`. It also seems that the `ExtLanguage` database does not fill up (it's empty on my local server), which might be why using a `get` fails.